aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -1590,13 +1591,9 @@ module aptos_framework::delegation_pool {
         current_delegated_votes.pending_inactive_shares = current_delegated_votes.pending_inactive_shares - shares_to_redeem;
     }
 
+    #[deprecated]
     public fun multiply_then_divide(x: u64, y: u64, z: u64): u64 {
-        let result = (to_u128(x) * to_u128(y)) / to_u128(z);
```

Comment:
Let's may be still keep the same code here in case someone is using this function on chain for whatever reasons

Reply:
lemme call `math64::mul_div` instead and add comments